theme/styles/abstracts/_variables.scss

```diff
@@ -67,7 +67,7 @@ $header: (
   layout-breakpoint: 1625px,
   breakpoint-for-search-bar: 850px,
   nav-link-padding-lateral: 1em,
-  nav-width-desktop: 600px,
+  nav-width-desktop: 450px,
```

Comment:
600px width was a change I made to reflect design of desktop nav's dropdowns, but on closer inspection I realized that if the rightmost top level nav item had a dropdown, that on most desktop screen sizes it would overflow out the right side of the body container.